src/core/reactor.cc

```diff
@@ -4516,7 +4516,7 @@ void smp::configure(const smp_options& smp_opts, const reactor_options& reactor_
 #ifdef SEASTAR_HAVE_DPDK
     if (_using_dpdk) {
         auto it = _thread_loops.begin();
-        RTE_LCORE_FOREACH_SLAVE(i) {
+        RTE_LCORE_FOREACH_WORKER(i) {
```

Comment:
Let's defer this patch until the thing is merged.

Reply:
sure. dropped.

Reply:
turns out i have to include it to get the change build with dpdk v23.07. so merged it into the first commit.